packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -34,17 +35,18 @@ const isSelected = () =>
 </script>
 
 <template>
-	<asset-card
+	<AssetCard
 		:asset="asset"
 		:resourceType="resourceType"
 		:active="isPreviewed"
+		:highlight="searchTerm"
 		@click="emit('toggle-asset-preview')"
 	>
 		<button type="button" @click.stop="emit('toggle-selected-asset')">
-			<i class="pi pi-plus" v-show="!isSelected()"></i>
-			<i class="pi pi-check checkmark-color" v-show="isSelected()"></i>
+			<i class="pi pi-plus" v-show="!isSelected()" />
```

Comment:
Just curious, is there any difference between doing this and doing ```<button icon=pi pi-plus>``` ?

Reply:
It shouldn't

Reply:
Makes sense!